torch/utils/cpp_extension.py

```diff
@@ -2084,6 +2124,15 @@ def sanitize_flags(flags):
         object_file = object_file.replace(" ", "$ ")
         build.append(f'build {object_file}: {rule} {source_file}')
 
+    if cuda_dlink_post_cflags:
+        devlink_out = os.path.join(os.path.dirname(objects[0]), 'dlink.o')
```

Comment:
Is there a risk of file name collision here?

Reply:
fixed